.github/workflows/release.yml

```diff
@@ -103,6 +104,35 @@ jobs:
           fi
           git tag -l -n --format="%(contents:subject)" $(git describe --tags --abbrev=0) > TAG-COMMENT.txt
 
+      - name: MSI
+        shell: bash
+        if: matrix.os == 'windows-latest'
+        run: |        
+          # Return a qldb.exe
+          cargo fetch && cargo build --release  
+          
+          strip "target/${{ matrix.target }}/release/${{ matrix.bin }}"
+          
+          cargo install cargo-wix
```

Comment:
My understanding is this command will create a default `main.wxs` file. Since you are committing a customized `main.wsx` file, do we still need to run this?

Reply:
I believe we need to install it so we can run the 'cargo wix' command that compiles the wxs file into an msi file.  We do not create a default main.wxs since that is supplied now by us.  The command that did that was cargo wix init.

Reply:
Can you remove comment in L117/118 to reflect this

Reply:
Removed